### ISSUE TITLE
Accessibility: Textarea: The ‘Help’ tooltip is not adapting to high contrast black theme

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3443]
Description: The ‘Help’ tooltip is not adapting to high contrast black theme

Expected: Ideally, each & every UI should be adaptable to high contrast theme

Attached screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94519594-7217df00-0248-11eb-94fd-11e2b2ce856a.png)

**Comment by kavya-b** [comment]
This has to be handled at the Platform Forms level, and is not related to Textarea.

**Comment by kavya-b** [comment]
@puru-hk this doesn't seem to be reproducing anymore on netlify and main url. Could you please verify?

**Comment by puru-hk** [comment]
This is issues is working fine.
